plugins/BEdita/API/src/Controller/ObjectsController.php

```diff
@@ -108,6 +101,44 @@ public function initialize()
         }
     }
 
+    /**
+     * Init model related attributes:
+     *  - $this->objectType
+     *  - $this->modelClass
+     *  - $this->Table
+     *
+     * @return void
+     * @throws \Cake\Routing\Exception\MissingRouteException If `object_type` param is not valid
+     */
+    protected function initObjectModel()
+    {
+        $type = $this->request->getParam('object_type', Inflector::underscore($this->request->getParam('controller')));
+        try {
+            /** @var \BEdita\Core\Model\Entity\ObjectType $this->objectType */
```

Comment:
[minor] This annotation may be omitted: class member is declared and documented at lines 52-57.

Reply:
Agreed, but `Scrutinizer` is complaining probably because we are using `TableRegistry::get('..')->get()`

Seems not solved though....